mappings/1.3-pre-07261249#1.3.1.tinydiff

```diff
@@ -2043,19 +2043,19 @@ c	net/minecraft/unmapped/C_5556017	net/minecraft/unmapped/C_5556017
 		p	1		p_1	
 c	net/minecraft/unmapped/C_5602044	net/minecraft/unmapped/C_5602044	
 	m	()V	<init>	<init>	
-c	net/minecraft/unmapped/C_5963566	net/minecraft/unmapped/C_5963566	
-	f	Ljava/util/Map;	f_9911567	f_9911567	
-	f	Lnet/minecraft/unmapped/C_4286596;	f_7834019	f_7834019	
-	f	Lnet/minecraft/unmapped/C_7699185;	f_7710270	f_7710270	
+c	net/minecraft/unmapped/C_5963566	net/minecraft/inventory/ContainerBackup	
```

Comment:
"Container" is not really used in Feather, this ought to be `InventoryBackup` then.